host-bmc/host_pdr_handler.cpp

```diff
@@ -173,6 +173,54 @@ HostPDRHandler::HostPDRHandler(
                 }
             }
         });
+
+    chassisOffMatch = std::make_unique<sdbusplus::bus::match::match>(
+        pldm::utils::DBusHandler::getBus(),
+        propertiesChanged("/xyz/openbmc_project/state/chassis0",
+                          "xyz.openbmc_project.State.Chassis"),
```

Comment:
does the chassis power solve all the cases...what about the host state ?

Reply:
PHYP leaves the slots on on a power off, so they will stay on until the hardware is turned off, which is the chassis.  We could maybe convince ourselves that host state would also work, but leaving it at chassis should be fine too.

Reply:
Ack